README.md

```diff
@@ -1,37 +1,29 @@
 # PackageListTool
 
-This tool allows creating package list YML files. It performs the following steps:
+This tool allows creating package list YML files. It has two subcommands:
 
-1. Fetch package summary descriptions from OpenAI's API for a list of package IDs (in this context: owner/repo pairs).
-2. These summaries are stored in a given directory at paths `outdir/owner-repo.txt` and will not be regnerated if a file already exists with that name.
-3. Fetch package metadata from the SPI API and merge it with the package summary.
+- `generate-descriptions`
+- `generate-packages-yml`
 
-Steps 1+2:
+`generate-descriptions` fetches package summary descriptions from OpenAI's API for a list of package IDs (in this context: owner/repo pairs). These summaries are stored in a given directory at paths `outdir/owner-repo.txt` and will not be regnerated if a file already exists with that name.
```

Comment:
`outdir` makes me think it's going to store in the cwd rather than the `./descriptions` directory. I know that directory hasn't been introduced yet, but thought I'd mention it as a minor thing.